src/chunk.c

```diff
@@ -4084,6 +4084,13 @@ ts_chunk_is_compressed(const Chunk *chunk)
 	return ts_flags_are_set_32(chunk->fd.status, CHUNK_STATUS_COMPRESSED);
 }
 
+bool
+ts_chunk_is_uncompressed_or_unordered(const Chunk *chunk)
+{
+	return (ts_flags_are_set_32(chunk->fd.status, CHUNK_STATUS_COMPRESSED_UNORDERED) ||
+			!ts_flags_are_set_32(chunk->fd.status, CHUNK_STATUS_COMPRESSED));
+}
+
```

Comment:
It's a matter of opinion, but I don't think it is necessary to have a separate function for anything as simple as this, it is quite clear as it is.